src/modes/hip/utils.cpp

```diff
@@ -151,31 +151,21 @@ namespace occa {
                        << "] does not support unified memory prefetching");
     }
 
-    hipCtx_t getContext(occa::device device) {
-      return ((hip::device*) device.getModeDevice())->hipContext;
-    }
-
-    void* getMappedPtr(occa::memory mem) {
-      hip::memory *handle = (hip::memory*) mem.getMHandle();
-      return handle ? handle->mappedPtr : NULL;
-    }
-
     occa::device wrapDevice(hipDevice_t device,
-                            hipCtx_t context,
                             const occa::properties &props) {
 
-      occa::properties allProps = props;
+      occa::properties allProps;
       allProps["mode"]     = "HIP";
-      allProps["device_id"] = -1;
+      allProps["deviceID"] = -1;
```

Comment:
I'm trying to stick with `snake_case` for property keys and `kebab-case` for command-like args.
```suggestion
      allProps["device_id"] = -1;
```

Reply:
Good catch. I thought I saw this change in the cuda backend and I must have missed these.